tests/fixtures/tests.py

```diff
@@ -286,6 +287,32 @@ def test_dumpdata_with_file_output(self):
         self._dumpdata_assert(['fixtures'], '[{"pk": 1, "model": "fixtures.category", "fields": {"description": "Latest news stories", "title": "News Stories"}}, {"pk": 2, "model": "fixtures.article", "fields": {"headline": "Poker has no place on ESPN", "pub_date": "2006-06-16T12:00:00"}}, {"pk": 3, "model": "fixtures.article", "fields": {"headline": "Time to reform copyright", "pub_date": "2006-06-16T13:00:00"}}]',
                 filename='dumpdata.json')
 
+    def test_dumpdata_progressbar(self):
+        """
+        Dumpdata shows a progress bar on the command line when --output is set,
+        stdout is a tty, and verbosity > 0.
+        """
+        management.call_command('loaddata', 'fixture1.json', verbosity=0)
+        new_io = six.StringIO()
+        new_io.isatty = lambda: True
+        filename = os.path.join(tempfile.gettempdir(), 'dumpdata.json')
```

Comment:
I think the filename isn't important so I would just use `_, filename = tempfile.mkstemp()`. Since temporary files are created inside a django_XXX directory by runtests.py, I don't think cleaning it up is necessary.